content/instructor/getting-started/onboarding/index.mdx

```diff
@@ -0,0 +1,56 @@
+---
+date: 2020-04-29
+title: 'Instructor Cohort Timeline'
+description: 'This article describes the onboarding process for new instructors.'
+categories: ['instructor', 'getting-started']
+published: true
+shareImage: 'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1571260698/og-image-assets/share_image_getting_started.png'
+---
+
+We start a new instructor cohort about every month. It will last for the full month, and the goal is to get you up to speed as a published egghead instructor over the month.
```

Comment:
I generalized the language so it's more informative on the whole process and took the CTA out.